src/Integration.Vsix/packages.config

```diff
@@ -1,4 +1,31 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="SonarLint" version="1.7.0" targetFramework="net452" developmentDependency="true" />
+  <package id="Microsoft.CodeAnalysis" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.1.0" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Common" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.CSharp" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic.Workspaces" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.1.1" targetFramework="net46" />
+  <package id="Microsoft.Composition" version="1.0.27" targetFramework="net46" />
+  <package id="Microsoft.VisualStudio.LanguageServices" version="1.1.1" targetFramework="net46" />
+  <package id="SonarLint" version="1.7.0" targetFramework="net46" developmentDependency="true" />
```

Comment:
@dbolkensteyn all the project in the solution use SonarLint (version 1.7) to get on-the-fly feedback on new bugs and quality issues injected into our code. You are right we are not forced to use it as we probably all have SonarLint installed as a VSIX.